package.json

```diff
@@ -35,16 +35,15 @@
     "test.watch": "yarn test --watch"
   },
   "peerDependencies": {
-    "@stoplight/markdown-viewer": "^4",
-    "@stoplight/ui-kit": "^3",
+    "@stoplight/markdown-viewer": "^5.0.0-beta.4",
+    "@stoplight/mosaic": "^1.0.0-beta.59",
```

Comment:
I mostly left it here because that's what we seemed to do before w markdown-viewer and ui-kit. I don't have a strong opinion - you want to leave as is or move them to deps?

Reply:
I started a discussion [on Slack](https://stoplight-internal.slack.com/archives/C80UNDN2E/p1622626495089600) although the conclusion was that pretty much neither is a great approach. @wmhilton suggests pulling the state out to its own package, that could solve the main problem and also fix the case of ESM vs CJS. But for the short term, IDK, whichever you prefer I guess. I'm good with it staying as is.